src/external/discord-rpc/include/discord_register.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
Did you try adding this as a module? This might just be github glitching out, but it looks like a hard copy to me.

Reply:
I'm not really sure what you mean here, could you elaborate please?

Reply:
he meant submodule: https://github.com/discordapp/discord-rpc

Reply:
and yes, do use it instead of the hard-copy